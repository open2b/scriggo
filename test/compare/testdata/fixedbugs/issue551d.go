// skip : expecting 'main.Iface', got 'Iface'

// paniccheck

package main

type Iface interface{}

func main() {
	var i Iface
	_ = i.(Iface)
}
