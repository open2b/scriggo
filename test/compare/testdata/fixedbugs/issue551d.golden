panic: interface conversion: interface is nil, not main.Iface